runtime/src/bank.rs

```diff
@@ -4669,7 +4674,8 @@ impl Bank {
             self.rent_collector.rent.burn_percent = 50; // 50% rent burn
         }
 
-        if new_feature_activations.contains(&feature_set::full_inflation::id()) {
+        if !new_feature_activations.is_disjoint(&self.feature_set.full_inflation_features_enabled())
```

Comment:
The if condition change here means that the block could be run multiple times, but I think that's ok